lib/shopify_app/sessions_concern.rb

```diff
@@ -19,6 +19,7 @@ def callback
         session[:shopify] = ShopifyApp::SessionRepository.store(sess)
         session[:shopify_domain] = shop_name
 
+        FulfillmentServicesManager.queue(shop_name, token) if ShopifyApp.configuration.has_fulfillment_services? && ShopifyApp.configuration.has_base_url?
```

Comment:
sure, this is a hangover from me injecting webhooks automatically on fulfilment service creation (hint: it doesn't work because async so the code is gone except this one artefact)